react/ProductSummaryReference.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import { ProductSummaryContext } from 'vtex.product-summary-context'
+
+const { useProductSummary } = ProductSummaryContext
+
+const CSS_HANDLES = ['referenceContainer']
```

Comment:
This array is readonly
```suggestion
const CSS_HANDLES = ['referenceContainer'] as const
```

Reply:
fixed!